.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "schemas"]
 	path = schemas
 	url = https://github.com/open-feature/schemas.git
+[submodule "jsonschemas"]
+	path = jsonschemas
+	url = https://github.com/open-feature/schemas.git
```

Comment:
Note, to accomplish this cleanly, I have cloned the schemas module twice (at different versions) one is used for json releases (and tagged to something like json/json-schema-v0.1.2) while the other is for the protobuf generation and tagged to a release of that. I don't love this, but I think it's needed.

Reply:
I've removed this 2nd submodule.